libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -126,9 +126,16 @@ class YulUtilFunctions
 
 	/// @returns the name of a function that rounds its input to the next multiple
 	/// of 32 or the input if it is a multiple of 32.
+	/// Ignores overflow.
 	/// signature: (value) -> result
 	std::string roundUpFunction();
 
+	/// @returns the name of a function that divides by 32 and rounds up during the division.
+	/// In other words, on input x it returns the smallest y such that y * 32 >= x.
+	/// Ignores overflow.
+	/// signature: (x) -> y
```

Comment:
Align with implementation?
```suggestion
	/// signature: (value) -> result
```

Reply:
I used x and y because this is easier to read in the text description above.